src/inputFile_impl.h

```diff
@@ -118,7 +118,8 @@ InputFile<t_System>::InputFile( InputCL commandline_, t_System *system_ )
     thermo_rate = 10;
 
     neighbor_skin = 0.3;
-    neighbor_skin = 0.0; // for metal and real units
+    // neighbor_skin = 0.0; // for metal and real units
```

Comment:
This double assignment was weird in the first place.  Why did you comment instead of removing the line?

Reply:
Agreed - I'll remove and document for that case somewhere